### ISSUE TITLE
Accesibility Issue: VSCode Notebook Plugin doesn't allow me to select the text that I'm reading.

### ISSUE THREAD

**Issue by BrainSlugs83** [dotnet/interactive#2157]
### Describe the bug
When using the ".NET Interactive Notebooks" plugin in VS Code, I cannot select text from markdown sections while I am reading them (like you can in any other program or web page). -- Instead it acts like it's trying to drag an image of the text somewhere. -- This is not intuitive behavior, and it's not the behavior of other Notebook readers such as Jupyter Notebooks.

I believe this is a major accessibility issue because, for people like me who have Visual Processing Disorders, this makes it hard to follow along with the text that I am reading as I read it.

### Please complete the following:

**Which version of .NET Interactive are you using?** (In a notebook, run the `#!about` magic command. ):
```
Version: 1.0.331401+f9a0736070089a9e182a96afc9c0142d2b81e5f5
Library version: 1.0.0-beta.22314.1+f9a0736070089a9e182a96afc9c0142d2b81e5f5
Build date: 2022-07-19T21:14:56.2564981Z
```

- OS
    - [ ] Windows 11
    - [x] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [x] Chrome
    - [ ] Edge
    - [ ] Firefox
    - [ ] Safari
 - Frontend
    - [ ] Jupyter Notebook
    - [ ] Jupyter Lab
    - [ ] nteract
    - [x] Visual Studio Code 
    - [ ] Visual Studio Code Insiders
    - [ ] Visual Studio
    - [ ] Other (please specify)

## Observed Behavior:

![dotnet-notebook](https://user-images.githubusercontent.com/5217366/179853644-d4052b85-83b5-4111-8bd5-0ccd2dc2a319.gif)

## Expected Behavior:

![jupyter-notebook](https://user-images.githubusercontent.com/5217366/179854375-60c61c05-f8f2-45b7-afc1-901205ff9c96.gif)

**Comment by AbhitejJohn** [comment]
Tagging @rebornix on this.
